src/core/cgroup.c

```diff
@@ -2314,10 +2304,12 @@ unsigned manager_dispatch_cgroup_realize_queue(Manager *m) {
         return n;
 }
 
-static void unit_add_siblings_to_cgroup_realize_queue(Unit *u) {
+void unit_add_siblings_to_cgroup_realize_queue(Unit *u) {
         Unit *slice;
+        Unit *self = u;
```

Comment:
doesn't matter much, but this looks backwards. i.e. instead of keeping a copy of the original pointer we shouldn't even modify the original pointer but a copy of it. i.e. intrdouce `Unit *p = u;` here, and then keep updating `p` instead of `u` when iterating through the tree.

Reply:
Thanks, avoided that with the family rework (but used that in the intermediate commits).